fre/make/createCompile.py

```diff
@@ -31,11 +31,25 @@ def compile_create(yamlfile,platform,target,jobs,parallel,execute,verbose):
     plist = platform
     tlist = target
 
+    ## If combined yaml does not exist, combine model, compile, and platform yamls
+    cd = Path.cwd()
+    combined = Path(f"combined-{name}.yaml")
+    combined_path=os.path.join(cd,combined)
+
+    # Combine model, compile, and platform yamls
+    # If fre yammltools combine-yamls tools was used, the combined yaml should exist
+    if Path(combined_path).exists():
+        full_combined = combined_path
+        print("\nNOTE: Yamls previously merged.")
+    else:
+        comb = cy.init_compile_yaml(yml,platform,target)
+        full_combined = cy.get_combined_compileyaml(comb)
```

Comment:
You should wrap this in a function so it can be reused instead of reusing this same code over and over.